package.json

```diff
@@ -7,14 +7,14 @@
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "axios": "^0.19.2",
-    "bootstrap": "^4.5.0",
+    "bootstrap": "^3.4.1",
```

Comment:
Please do not downgrade the bootstrap. We are using bootstrap 4 and not 3. Revert this change.

Additionally please remove package-lock.json. this project uses Yarn and this is resulting in build failures